### PR TITLE
Allow whitespace and support multi-line providers

### PR DESCRIPTION
**Pull request by timbru**

### CODE REVIEW

File: draft-ietf-sidrops-aspa-notation-02.md [hunk 1]
```diff
@@ -84,14 +84,27 @@ to use their own notation styles instead of, or in addition to this.
...
-asn                = ["AS"] uint32
-uint32             = %d0-4294967295
+notation            = customer-asid separator providers
+
+customer-asid       = asn
+separator           = " => "
+
+providers           = providers-one-line / providers-multiline
+providers-one-line  = asn *(*wsp "," *wsp asn)
+providers-multiline = "[" *wspml asn *(*wspml "," *wspml asn) *wsp "]"
```
Comment (ties) [inline draft-ietf-sidrops-aspa-notation-02.md:94]: Does `wsp` allow the final newline in examples like 
```
AS65000 => [
    AS65001,
    AS65002,
    AS65003
]
```?

File: draft-ietf-sidrops-aspa-notation-02.md [hunk 2]
```diff
@@ -118,21 +131,34 @@ This field represents a Provider AS as defined in section 3.3 of
...
+AS65000 => [ AS65001, AS65002, AS65003 ]
+
+AS65000 => [
+    AS65001,
+    AS65002,
+    AS65003
+]
+
+AS65000 => [AS65001,
+                     65002
```
Comment (ties) [inline draft-ietf-sidrops-aspa-notation-02.md:159]: No longer allowed?

**Review by ties — APPROVED** [review]
Two nits, looks good otherwise!

File: draft-ietf-sidrops-aspa-notation-02.md (hunk 1 above)
Comment (timbru) [reply draft-ietf-sidrops-aspa-notation-02.md:94]: Indeed - this should be wspml (also mentioned in the list somewhere or in direct email to me).

But, I will merge this first and then make a new branch for -03 and fix it there.

File: draft-ietf-sidrops-aspa-notation-02.md (hunk 2 above)
Comment (timbru) [reply draft-ietf-sidrops-aspa-notation-02.md:159]: Indeed - this should be AS65002

But, I will merge this first and then make a new branch for -03 and fix it there.

**Pull request merged**